source/jormungandr/jormungandr/interfaces/v1/Journeys.py

```diff
@@ -458,6 +458,18 @@ def __init__(self):
             hidden=True,
             help="Here, Max number of matrix points for the street network computation (limited to 100)",
         )
+        parser_get.add_argument(
+            '_sn_exclusion_area[]',
+            type=six.text_type,
+            case_sensitive=False,
+            hidden=True,
+            action='append',
+            dest='_sn_exclusion_areas',
```

Comment:
If I'm not mistaken, this parameters applies only to the car fallbacks obtained with Here. But the name  `sn_exclusion_areas` suggests it applies to all street network computations.
Maybe rename it to something more specific ? Like "here_sn_exclusion_areas" ?

Reply:
i didn't think about that. I accept your change, it will be more understandable